nixos/modules/system/boot/systemd/coredump.nix

```diff
@@ -0,0 +1,58 @@
+{ config, lib, pkgs, utils, ... }:
+
+with lib;
+
+let
+  cfg = config.systemd.coredump;
+  systemd = config.systemd.package;
+in {
+  options = {
+    systemd.coredump.enable = mkOption {
```

Comment:
```suggestion
  options.systemd.coredump = {
    enable = mkOption {
```

Reply:
Debatable change but if done should also done to the other modules

Reply:
Meh, most files followed the `options = {` style approach, so I followed by example. It's fine to change them all and I'm fine to do that, but it seems more fitting in a separate PR.

If there is an unwritten rule that `options.* =` style approach should be taken, let's write that down somewhere. (linting rules?)